support/troubleshooting/troubleshooting-network-issues.adoc

```diff
@@ -0,0 +1,10 @@
+[id="troubleshooting-network-issues"]
+= Troubleshooting network issues
+
+include::modules/common-attributes.adoc[]
+:context: troubleshooting-network-issues
+
+toc::[]
+
```

Comment:
Is the `toc::[]` not working? I don't see a [TOC in the preview](https://deploy-preview-32693--osdocs.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-network-issues.html).

I think it is the carriage return above the include::modules/common-attributes.adoc[]